src/plugins/maps_dashboards/common/index.ts

```diff
@@ -19,9 +19,11 @@ export const APP_PATH = {
   CREATE_MAP: '/create-map',
 };
 
-export enum LAYER_TYPE {
+export enum MAP_LAYER_TYPE {
```

Comment:
Any reason to rename this variable? i like Layer type .

Reply:
I want to separate the layer conception in code from our layer and maplibre layer, so that we can call ours as `mapLayer ` and maplibre's as `mbLayer`

Reply:
May be DASHBOARDS_MAPS_LAYER_TYPE?

Reply:
Updated